heon/multi_client.py

```diff
@@ -0,0 +1,101 @@
+import socket
+from FlowRecord import FlowRecord
+import argparse
+
+# Define constants
+MAX_CHUNK_SIZE = 100
+
+# Initialize 3 client UDP sockets
+cs_list = []
+
+# Empty previous flow_sequence in flie
+f = open("flow_sequence_client_side.txt", "w")
+f.write("")
+f.close()
+
+# Empty previous ts in file
+f = open("flow_ts_client_side.txt", "w")
+f.write("")
+f.close()
+
+
+# Create a new client socket
+def init_client_socket():
+    cs = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+    return cs
+
+
+# Run a single client connection. Send a msg to the UDP server and
+# listen for a response.
+def run_single_client_connection(cs, flow_num, server_addr):
+    print(f"sending: {flow_num}")
+    flow_record = FlowRecord(flow_num=flow_num)
+
+    cs.sendto(str.encode(str(flow_record.getFlowNum())), server_addr)
+    print("sent!")
+
+    f = open("flow_sequence_client_side.txt", "a")
```

Comment:
I'm assuming these are sanity checks to ensure that the input order set in the `input_flow_seq_file` is the same as the one you transmitted in. Correct? If so, please add a short comment to state that. 

Btw you could define these filenames in arguments with defaults, too, to make the code cleaner.

Reply:
Yes, the print statements are just to see the flow numbers being transmitted to make sure the order is correct. I will add a comment to explain that. 

I forgot to add arguments for the filenames, thanks for catching this!